interfaces/Portalicious/src/app/components/registration-page-layout/registration-page-layout.component.ts

```diff
@@ -137,6 +144,21 @@ export class RegistrationPageLayoutComponent {
         ),
       ],
     },
+    {
+      label: $localize`:@@registration-duplicates:Duplicates`,
+      items: [
+        {
+          label: $localize`:@@ignore-duplication:Ignore duplication`,
+          icon: 'pi pi-clone',
+          visible:
```

Comment:
I wonder whether this should be visible only with a specific permission? Maybe not